src/server/lib/passcodeSendAgainTimer.ts

```diff
@@ -0,0 +1,15 @@
+import { Request } from 'express';
+
+export const getPasscodeSendAgainTimer = (req: Request): number => {
+	// if we're running in cypress, we want to use the cypress mock state cookie to set the timer for testing
+	const runningInCypress = process.env.RUNNING_IN_CYPRESS === 'true';
+	if (runningInCypress) {
+		const cypressMockStateCookie = req.cookies['cypress-mock-state'];
```

Comment:
Some questions to understand the cypress tests better:

- does `cypress-mock-state` hold the value for the counter specifically  or it just a value used for anything ?
- Where is it set ?